dkulib/dku_config/custom_check.py

```diff
@@ -109,7 +109,7 @@ def _exists(self, value: Any) -> bool:
         Returns:
             bool: Whether the check has succeed
         """
-        return value is not None
+        return value is not None and value != []
```

Comment:
If we add `[]` to be considered as empty, we should also add `""`. 
```suggestion
        EMPTY_ENTRIES = [[], "", None]
        return value not in EMPTY_ENTRIES
```